.readthedocs.yaml

```diff
@@ -16,9 +16,13 @@ formats:
   - htmlzip
   - pdf
 
+build:
```

Comment:
Can you convert this to using mamba like astropy uses.

Reply:
Sure, but that's probably for another PR as it would also require an environment file

Reply:
Well the environment you targeted in this PR is causing the docs to fail.